src/app/models/types/curationData.ts

```diff
@@ -67,3 +67,10 @@ export interface Curation extends BaseCuration {
   curationLength?: number;
   nthCurationByStyleAndProminence?: number;
 }
+
+export interface BillboardSummary extends BaseSummary {
+  imageUrl: string;
+  link: string;
+  description: string;
+  imageAlt: string;
+}
```

Comment:
As mentioned above, I don't think we need this anymore - this can be handled with destructuring and other TS methods.